src/controllers/teamController.spec.js

```diff
@@ -0,0 +1,90 @@
+const Team = require('../models/team');
+const teamController = require('./teamController');
+const { mockReq, mockRes, assertResMock } = require('../test');
+const helper = require('../utilities/permissions');
+
+const makeSut = () => {
+  const { getAllTeams, getTeamById, postTeam } = teamController(Team);
+  return {
+    getAllTeams,
+    getTeamById,
+    postTeam,
+  };
+};
+
+jest.mock('../utilities/permissions', () => ({
```

Comment:
This line of code is unnecessary. `mockHasPermission` already mocks the `hasPermission` function.